### PR TITLE
Updated browser docs

### PR DESCRIPTION
**Pull request by RangerMauve**
Updated the docs to reflect the changes in dat-js.

I'm open ideas for what could be added to the docs.

### CODE REVIEW

**Comment by RangerMauve** [comment]
K, I think this is good to go. @karissa @joehand

File: docs/cookbook/browser.md [hunk 1]
```diff
@@ -14,33 +14,45 @@ OK, now for the goods.
...
+And build a bundle with browserify, assuming your file is called `index.js` like this:
+
+```
+browserify ./index.js > bundle.js
+```
+
+Then include a script tag pointing at your `bundle.js`:
+
+```
+<scrupt type="text/javascript" src="bundle.js"></script>
```
Comment (karissa) [inline docs/cookbook/browser.md]: typo

Comment (RangerMauve) [reply docs/cookbook/browser.md]: Oops. 😅

**Comment by joehand** [comment]
Thanks! I turned off netlify build for the old site. Let's get the [new site](https://github.com/datproject/docs/pull/142) out by the end of the weekend! 

@RangerMauve I added your edits in my PR too.

**Comment by RangerMauve** [comment]
@joehand I'll close this in favor of your issue. 😀

**Pull request closed without merging**